src/posts/fsweekend-2024.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'FlyByWire Simulations at FS Weekend 2024'
+date: '2024-04-07'
+authors:
+  - 'straks'
+category: 'INFO'
+metaImage: '/img/notam-images/fsweekend2024/header.png'
+metaAlt: 'FlyByWire Simulations at FS Weekend 2024'
+---
+
+A few weeks ago, over the weekend of March 16th and March 17th, 2024, the [FS Weekend](https://fsweekend.nl/) conference took place in the [Aviodrome Museum](https://www.aviodrome.nl/) in Lelystad in the Netherlands. This conference brings together Flight Simulation enthusiasts and professionals from around the world. The Aviodrome Museum presents the perfect environment for a flight simulation event with its rich presentation of the history of aviation through several exhibits and a large range of aircraft from different time periods.
+
+FlyByWire Simulations was present at FS Weekend with a large booth and fourteen of our most dedicated developers and team members. We welcomed hundreds of fans to our booth who got to try out flying the A32NX and our new project, the A380X.
+
+## Our Booth and Team
+
+For the FlyByWire Simulations team, events like this are the perfect time to meet up with each other in person and build our personal bonds even more, while also showing what an Free and Open Source developer group can achieve when working together on a passion in the flight simulation community.
+
+As a team, we worked on designing a booth with great marketing material and graphics, making sure everything is in our branding style. We brought together skills and people from around the world to set up a booth that looked amazing and inviting to our visitors.
```

Comment:
If possible coudl you expand upon the marketing materials here? It would be good to state like our digital displays, brochures, banners, etc. Since a lot of people worked on stuff to get printed and itemizing it would give a nice little "credit" on what was brought over.